extensions/amp-bind/amp-bind.md

```diff
@@ -291,7 +294,7 @@ An `amp-state` element must contain a child `<script>` element.
 
 ##### `src` (optional)
```

Comment:
header too deep?

Reply:
Nope. Usually Attributes gets a top heading, but this is specific to the `<amp-state>` component, which is not needed to use `amp-bind`. I guess it's a technicality?